setup.py

```diff
@@ -0,0 +1,31 @@
+import setuptools
+import pathlib
+from dj_search.meta import pkg_name, __version__
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+with open(pathlib.Path(__file__).parent / 'dj_search' / 'meta.py') as f:
+    exec(f.read())
+
+with open(pathlib.Path(__file__).parent / 'requirements.txt') as f:
+    requirements = f.read().splitlines()
+
+setuptools.setup(
+    name=pkg_name,
+    version=__version__,
+    author="Thinh Nguyen",
+    author_email="thinh@vathes.com",
+    description="Unofficial search utility for DataJoint pipeline.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/datajoint/dj-search",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+    ],
+    install_requires=requirements,
+    entry_points={},
+)
```

Comment:
Got an error when I pip install in my conda env. Not sure about why, but maybe related to the setup.py

```
    ERROR: Command errored out with exit status 1:
     command: /Users/shanshen/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/kb/zd2r0th56mxf_hchvswnnv_c0000gn/T/pip-req-build-gxkp_1_6/setup.py'"'"'; __file__='"'"'/private/var/folders/kb/zd2r0th56mxf_hchvswnnv_c0000gn/T/pip-req-build-gxkp_1_6/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/kb/zd2r0th56mxf_hchvswnnv_c0000gn/T/pip-pip-egg-info-c7661kgo
         cwd: /private/var/folders/kb/zd2r0th56mxf_hchvswnnv_c0000gn/T/pip-req-build-gxkp_1_6/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/Users/shanshen/anaconda3/lib/python3.7/tokenize.py", line 447, in open
        buffer = _builtin_open(filename, 'rb')
    FileNotFoundError: [Errno 2] No such file or directory: '/private/var/folders/kb/zd2r0th56mxf_hchvswnnv_c0000gn/T/pip-req-build-gxkp_1_6/setup.py'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

```